lib/ansible/modules/_apt_repository.py

```diff
@@ -16,6 +16,10 @@
 description:
     - Add or remove an APT repositories in Ubuntu and Debian.
 extends_documentation_fragment: action_common_attributes
+deprecated:
+    alternative: ansible.builtin.deb822_repository
+    why: The M(ansible.builtin.apt_repository) module is deprecated in favor of the M(ansible.builtin.deb822_repository) module.
+    removed_in: 2.24
```

Comment:
> lib/ansible/modules/_apt_repository.py:0:0: invalid-documentation: DOCUMENTATION.deprecated.removed_in: not a valid value for dictionary value @ data['deprecated']['removed_in']. Got 2.24

It requires a string

```suggestion
    removed_in: "2.24"
```